src/main/java/appeng/menu/me/items/CraftingTermMenu.java

```diff
@@ -78,8 +83,6 @@ public CraftingTermMenu(int id, final Inventory ip, final ITerminalHost host) {
         this.addSlot(this.outputSlot = new CraftingTermSlot(this.getPlayerInventory().player, this.getActionSource(),
                 this.powerSource, host, craftingGridInv, craftingGridInv, this), SlotSemantic.CRAFTING_RESULT);
 
-        this.createPlayerInventorySlots(ip);
```

Comment:
I think the point behind this was that the player slots were always created last (because they *are* in order), but I honestly don't remember why that would be important.